.github/pull_request_template.md

```diff
@@ -18,10 +20,13 @@ What types of changes does your code introduce? Put an `x` in all the boxes that
 ## Checklist
  - [ ] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
  - [ ] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
+ - [ ] My code follows the style guidelines of this project
+ - [ ] I have performed a self-review of my code
  - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
- - [ ] I have implemented unit tests to cover my changes
  - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
  - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
+ - [ ] I have added tests that prove my fix is effective or that my feature works
+ - [ ] New and existing unit tests pass locally with my changes
```

Comment:
These changes are really good. Were they your idea or did you source them from somewhere else?

Reply:
I copied it from somewhere

Reply:
If you recall the source please share it. Maybe we can get inspiration for other ideas too :sweat_smile:

Reply:
I've updated PR description